src/v2/coins/abstractUtxoCoin.js

```diff
@@ -665,19 +665,20 @@ class AbstractUtxoCoin extends BaseCoin {
       debug('Input details: %O', currentSignatureIssue);
 
 
-      const isBech32 = !currentUnspent.redeemScript;
-      const isSegwit = !!currentUnspent.witnessScript;
+      const isP2wsh = !currentUnspent.redeemScript;
       const sigHashType = this.constructor.defaultSigHashType;
       try {
-        if (isBech32) {
+        if (isP2wsh) {
+          debug('Signing p2wsh input');
           const witnessScript = Buffer.from(currentUnspent.witnessScript, 'hex');
           const witnessScriptHash = bitcoin.crypto.sha256(witnessScript);
           const prevOutScript = bitcoin.script.witnessScriptHash.output.encode(witnessScriptHash);
           txb.sign(index, privKey, prevOutScript, sigHashType, currentUnspent.value, witnessScript);
         } else {
           const subscript = new Buffer(currentUnspent.redeemScript, 'hex');
-          if (isSegwit) {
-            debug('Signing segwit input');
+          const isp2shP2wsh = !!currentUnspent.witnessScript;
```

Comment:
`isP2shP2wsh`? (first p should be capitalized?)

Reply:
Good point. Fixed.